### ISSUE TITLE
How does DPI affect the model quality in Training? [question]

### ISSUE THREAD

**Issue by mrocr** [Calamari-OCR/calamari#71]
Does training a model with having each image scanned with `150, 200 & 300 dpi` improve the quality of the model?

**Comment by ChWick** [comment]
Each line is down scaled to a fixed line height of 48px (by default), therefore even if you have an image with a higher dpi the image is scaled to the same height.
A possible approach is to set the line height (parameter of train) to a value > 48, e. g. 100, to use images with a high dpi in full resolution, however in practice I do not expect the quality to improve since (especially for a human) all relevant information is clearly visible, even dots or dashes. I even expect that the model could worsen if one uses a large line height and only very few lines for training: The model will easily overfit on the increase input data amount.

**Comment by mrocr** [comment]
@ChWick
What are your thoughts on [this published paper](https://arxiv.org/abs/1812.11894), they seem to use very simple neural network architecture:
- They trained for 8 epochs & achieved CER of 0.08%. `90k lines for training and 10k for testing.`
- They used the input scale of 32 pixels.
- No text line normalization.

How were they able to achieve 0.08% cer with only 8 epochs and such small input scale without any preprocessing using a simple f-cnn? Am I missing something?

**Comment by ChWick** [comment]
@mrocr 
Thanks for the link!
* The best calamari model (0.114% CER) trained for approximately 3 epochs only (early stopping at iteration 200,000 using 70k lines for training)
* Calamari uses 48px line height. If I got some time, I will train calamari on this smaller line height, and post my results, however, since even with 32 px all letters are clearly visible I assume good results will be obtained.
* I never used the non-normalized lines for training. I will also test this. However, I would like to see their results using the normalized lines (maybe they are even better)

For our models we used a slightly different dataset. Most of the Calamari errors are errors of the dataset itself (GT errors). Since the error rates are so low, the quality of the dataset itself should be close to the resulting errors.
